File: app/src/main/java/com/dev/recyclica/MainActivity.java
package com.dev.recyclica;

import android.os.Bundle;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;

import com.dev.recyclica.fragments.AuthorizationFragment;

import moxy.MvpAppCompatActivity;

public class MainActivity extends MvpAppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            final FragmentManager manager = getSupportFragmentManager();
            final Fragment fragment = new AuthorizationFragment();
            manager.beginTransaction()
                    .add(R.id.activity_container, fragment, AuthorizationFragment.TAG)
                    .commit();
        }
    }
}

File: app/src/main/java/com/dev/greenday/MainActivity.kt
package com.dev.greenday

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.FragmentManager
import com.dev.greenday.fragment.FragmentAuthorization
import com.dev.greenday.fragment.FragmentMenu

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (savedInstanceState == null) {
            val fragment = FragmentAuthorization()
            supportFragmentManager.beginTransaction()
                .add(R.id.container, fragment, "authorization")
                .commit();
        }
    }
}
